chore: Update README (#203)
anaconda/anaconda-cli: README.md

> # Ana CLI

Reviewer: Do we want to prefer "Anaconda CLI" in a textual setting? Are we aligning w/ conventions for capitalization, similar to [conda's](https://docs.conda.io/projects/conda/en/stable/dev-guide/contributing.html#conda-capitalization-standards)?

Author: Changed to "Anaconda CLI" everywhere. Capital "A" makes sense since Anaconda is capitalized

A command-line tool for managing Anaconda tools, authentication, and package channels.

## When to use this

Ana is the entry point for the Anaconda ecosystem. It handles authentication to Anaconda services, configures tools like conda and pixi, and manages access to package channels, so you don't have to manually edit config files or manage credentials across tools separately.

> If you're building with Anaconda's data science, ML, or AI tooling, ana gets you from zero to a working environment in minutes.

Reviewer: ```suggestion
If you're building with Anaconda's data science, ML, or AI tooling, ana gets you from zero to a working environment in seconds.
```

Maybe a bit too ambitious for right now 😄 ?

## Project status

Ana CLI is under active development and follows semantic versioning. Commands not marked as experimental are considered stable. Features behind `ana feature enable` may change between releases.

## Prerequisites

> - macOS (Apple Silicon or x86_64), Linux (x86_64 or aarch64), or Windows (x86_64)

Reviewer 2: ```suggestion
- macOS (Apple Silicon), Linux (x86_64 or aarch64), or Windows (x86_64)
```

We don't produce Intel binaries for macOS. See also #167

- [conda](https://www.anaconda.com/download) installed if you plan to create conda environments
- Internet connection required on first run on macOS (Gatekeeper notarization check)

## Installation

### macOS and Linux

```bash
curl -fsSL https://anaconda.sh/install.sh | sh
```

### Windows (PowerShell)

```powershell
powershell -ExecutionPolicy ByPass -c "irm https://anaconda.sh/install.ps1 | iex"
```

### Installation options

> | Option | Environment variable | Default | Description |

Reviewer 2: Why were the PowerShell options removed?

|--------|---------------------|---------|-------------|
| `-d, --install-dir` | `ANA_INSTALL_DIR` | `~/.local/bin` | Installation directory |
| `-v, --version` | `ANA_VERSION` | `latest` | Specific version to install |

> | `--no-verify-checksum` | `ANA_VERIFY_CHECKSUM=false` | verify | Skip checksum validation |

Reviewer 2: ```suggestion
| `--no-verify-checksum` | `ANA_VERIFY_CHECKSUM` | verify | Skip checksum validation |
```

| `--no-path-update` | `ANA_NO_PATH_UPDATE` | update | Skip shell profile modification |

> | `-f, --force` | `ANA_FORCE_INSTALL` | prompt | Overwrite existing installation |

Reviewer 2: I think I get what those defaults are supposed to tell me, but when I see "default", I expect a default value,                  and those aren't the default values.

## Quick start

```bash
# 1. Verify ana is installed
ana --version

> # 2. Log in to your Anaconda account (opens browser)

Reviewer: ```suggestion
# 2. Log in to your Anaconda account (opens browser if available)
```

```

### MCP (Model Context Protocol)

```bash
ana mcp setup                    # Configure AI clients for Anaconda MCP
ana mcp discover                 # Discover MCP servers from environments
ana mcp clients                  # List supported AI clients
ana mcp serve                    # Start MCP servers
ana mcp remove                   # Remove MCP configuration
```

### API access

```bash
ana api fetch /api/account                                        # GET request
ana api fetch /api/endpoint --method POST --json '{"key": "val"}' # POST with JSON
ana api fetch /api/search -q "name=numpy,version=1.24"            # Query parameters
```

### Anaconda CLI (classic)

If you need the classic `anaconda` command (for uploading packages to anaconda.org, etc.):

```bash
ana bootstrap
```

This installs the `anaconda` command, which you can then use directly:

```bash
anaconda upload my-package.tar.bz2
anaconda search numpy
```

## Troubleshooting

### "Command not found" after installation

The installer adds `~/.local/bin` to your PATH via your shell profile. Restart your shell, or run:

```bash
export PATH="$HOME/.local/bin:$PATH"
```

### First run on macOS hangs or fails

> macOS Gatekeeper verifies the binary's notarization on first run, which requires an internet connection. If verification fails, go to System Settings > Privacy & Security and allow the binary to run.

Reviewer 2: Has this been verified?

Author: removed for now

### Login fails with "invalid API key"

> Verify your API key at [anaconda.cloud](https://anaconda.cloud). Keys are environment-specific (production vs. staging) and won't work cross-environment.

Reviewer: ```suggestion
Verify your API key at [anaconda.com](https://anaconda.com/app/profile/api-keys). 
Keys are environment-specific (anaconda.com vs. self-hosted) and won't work cross-environment.
```

### SSL certificate errors

If you're behind a corporate proxy or using custom CA certificates:

```bash
export ANA_SSL_VERIFY=false
ana login
```

Only disable SSL verification in trusted network environments.

### Verbose logging

```bash
ana -vvv login                   # Up to 5 verbosity levels
RUST_LOG=ana=debug ana login      # Fine-grained control via RUST_LOG
```

> ## Documentation

Reviewer: Do we want to put this above the README docs?

Author: errant, thank you

```

### Windows (PowerShell)

```powershell
powershell -ExecutionPolicy ByPass -c "irm https://anaconda.sh/install.ps1 | iex"
```

### Installation options

| Bash | PowerShell | Environment variable | Default | Description |
|------|------------|---------------------|---------|-------------|
| `-d, --install-dir` | `-InstallDir` | `ANA_INSTALL_DIR` | `~/.local/bin` | Installation directory |
| `-v, --version` | `-Version` | `ANA_VERSION` | `latest` | Version to install |
| `--no-verify-checksum` | `-NoVerifyChecksum` | `ANA_VERIFY_CHECKSUM` | `true` | Set to `false` to skip checksum validation |
| `--no-path-update` | `-NoPathUpdate` | `ANA_NO_PATH_UPDATE` | | Set to skip shell profile modification |
| `-f, --force` | `-Force` | `ANA_FORCE_INSTALL` | | Set to overwrite without prompting |

## Quick start

```bash
# 1. Verify installation
ana --version

# 2. Log in to your Anaconda account (opens browser if available)
ana login

# 3. Enable early-access packages from the main-x channel
ana feature enable main-x

# 4. Create your first environment
conda create -n myproject python=3.12 numpy pandas jupyter
conda activate myproject
```

## Common workflows

> ### Configuring pixi for Anaconda channels

Reviewer: ```suggestion
### Configuring conda and pixi for Anaconda channels
```

### Environment variables

| Variable | Default | Description |
|----------|---------|-------------|
| `ANA_DOMAIN` | `anaconda.com` | Authentication domain |
| `ANA_SSL_VERIFY` | `true` | Verify SSL certificates |
| `ANA_OPEN_BROWSER` | `true` | Auto-open browser during login |
| `ANA_ENABLE_TELEMETRY` | `true` | Send anonymous usage telemetry (see [telemetry policy](docs/telemetry.md)) |
| `ANA_PRERELEASES` | `false` | Include pre-releases in update checks |
| `ANA_KEYRING_PATH` | `~/.anaconda/keyring` | API key storage location |

Boolean values accept `true`/`false`, `1`/`0`, or empty string (treated as false).

### Feature flags

Experimental features are stored in `~/.ana/config.toml`:

```toml
[ana.features]
outerbounds = true
```

## Command reference

### Authentication

```bash
ana login                        # Interactive login (browser-based device flow)
ana login --api-key "your-key"   # Login with API key directly
echo "$KEY" | ana login --api-key  # Login with API key from stdin
ana whoami                       # View account info
ana whoami --json                # Account info as JSON
ana logout                       # Log out

> ```

Reviewer: ```suggestion
ana login                          # Interactive login (browser-based device flow)
ana login --api-key "your-key"     # Login with API key directly
echo "$KEY" | ana login --api-key  # Login with API key from stdin
ana whoami                         # View account info
ana whoami --json                  # Account info as JSON
ana logout                         # Log out
```
```
